website/landing/components/common/Clipboard.tsx

```diff
@@ -100,7 +100,7 @@ export const Clipboard: React.FC = () => {
           transition: "color .2s ease", // TODO: verify transition
           willChange: "color",
 
-          "&:hover, &:hover path": {
+          "&:hover": {
```

Comment:
Will the clipboard icon change its color without this?

Reply:
Yes, removing the color from the next element makes the SVG inherit color from here

Reply:
Great! I asked because it didn't work when I was implementing it. :/